Dockerfile

```diff
@@ -26,16 +26,11 @@ VOLUME [ /near ]
 WORKDIR /near
 COPY . .
 
-ENV CARGO_TARGET_DIR=/tmp/target
-ENV RUSTC_FLAGS='-C target-cpu=x86-64'
 ENV PORTABLE=ON
-ARG features=default
-RUN cargo build -p neard --release --features $features && \
-    mkdir /tmp/build && \
-    cd /tmp/target/release && \
-    mv ./neard /tmp/build
-
-COPY scripts/run_docker.sh /tmp/build/run.sh
+ARG make_target=
+RUN make CARGO_TARGET_DIR=/tmp/target \
+         RUSTC_FLAGS='-C target-cpu=x86-64' \
```

Comment:
This environment variable doesn't seem standard. I think this may be doing nothing and can be removed.

Reply:
By the way, do you know what `PORTABLE` might be?

Reply:
:shrug: but it isn't an environment variable rust toolchain inspects either.